pkg/apis/apps/v1/conversion.go

```diff
@@ -476,7 +478,8 @@ func Convert_v1_StatefulSetStatus_To_apps_StatefulSetStatus(in *appsv1.StatefulS
 
 func Convert_apps_StatefulSetStatus_To_v1_StatefulSetStatus(in *apps.StatefulSetStatus, out *appsv1.StatefulSetStatus, s conversion.Scope) error {
 	if in.ObservedGeneration != nil {
-		out.ObservedGeneration = *in.ObservedGeneration
+		out.ObservedGeneration = new(int64)
```

Comment:
`out.ObservedGeneration` is type `int64`. Should not create a new int64 ptr here.
Keep this unchanged.

Actually there are inconsistence between `apps/internal` and `apps/v1`. Such as `ObservedGeneration` is type `*int64` in `apps/internal`, while it is changed to `int64` in `apps/v1`.